newrelic/resource_newrelic_workflow.go

```diff
@@ -512,8 +512,7 @@ func resourceNewRelicWorkflowUpdate(ctx context.Context, d *schema.ResourceData,
 	accountID := selectAccountID(providerConfig, d)
 	updatedContext := updateContextWithAccountID(ctx, accountID)
 
-	// Setting to false to fix compilation errors
-	workflowResponse, err := client.Workflows.AiWorkflowsUpdateWorkflowWithContext(updatedContext, accountID, false, *updateInput)
+	workflowResponse, err := client.Workflows.AiWorkflowsUpdateWorkflowWithContext(updatedContext, accountID, true, *updateInput)
```

Comment:
`true`  represents the current behaviour before the client update.

`false` is the desired value, but I will create a separate PR to change it because it would change the behaviour I would thus need to change the tests and add a couple of new ones. 

I do want to mix the build fix with business changes.